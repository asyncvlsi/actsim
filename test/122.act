namespace pfloat {
    function f(int<4> x) : int<3> {
        chp {
            self := int(x{0}, 3)
        }
    }

    export
    defproc test() {
        int<15> x;
        chp {
            x := 123;
            log(f(x));
            log(int(7, 3) - f(x))
        }
    }
}

defproc test()
{
  pfloat::test t;
}
