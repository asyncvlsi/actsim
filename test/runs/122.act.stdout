[                  10] <t>  1
[                  10] <t>  6
